src/main/java/games/strategy/triplea/util/TuvUtils.java

```diff
@@ -138,14 +138,17 @@ private TuvUtils() {}
 
   private static int getTotalTuv(final UnitType unitType, final IntegerMap<UnitType> costs,
       final Set<UnitType> alreadyAdded) {
-    int tuv = costs.getInt(unitType);
     final UnitAttachment ua = UnitAttachment.get(unitType);
+    if (ua.getTuv() > 0) {
```

Comment:
My static analyzer is complaining here.  Do you need to check for a `null` unit attachment?  It appears you are doing so below on line 146.

Reply:
Yeah. I'll add a null check there.